test/test_reductions.py

```diff
@@ -2086,7 +2086,7 @@ def _compare_std_var_with_numpy(self, op, device, dtype, input, dim,
                 return
             self.fail("Failed to hit RuntimeError!")
 
-        exact_dtype = input.dtype != torch.bfloat16
+        exact_dtype = input.dtype not in (torch.bfloat16, torch.complex32, torch.complex64, torch.complex128)
```

Comment:
Both `torch` and `numpy` convert a complex input into a float. Thus, we either ignore these cases here or we need a better mechanism to create an out tensor with the expected dtype.

Reply:
Ignoring seems OK for now

Reply:
(Might as well add a TODO comment here with your thinking)